groups/live-windows-virtual-machines/abbyy_doc_ocr_ec2.tf

```diff
@@ -0,0 +1,175 @@
+    },
+    {
+      from_port   = 6129
+      to_port     = 6129
+      protocol    = "tcp"
+      cidr_blocks = "10.172.0.0/17"
+    },
+    {
+      from_port   = 49155
+      to_port     = 49155
+      protocol    = "tcp"
+      cidr_blocks = join(",", local.abbyy_doc_ocr_49155_cidr_block)
+    },
+    {
+      from_port   = 53853
+      to_port     = 53853
+      protocol    = "tcp"
+      cidr_blocks = "172.16.101.82/32"
+    },
+    {
+      from_port   = 49227
+      to_port     = 49227
+      protocol    = "tcp"
+      cidr_blocks = "172.16.101.82/32"
+    },
+    #{
+    #rule        = "rdp-tcp"
+    #cidr_blocks = join(",", local.abbyy_doc_ocr_third_party_rdp_cidr_block)
+    #}
+  ]
+
+  computed_ingress_with_source_security_group_id = [
+    {
+      rule                     = "http-80-tcp"
+      source_security_group_id = module.abbyy_doc_ocr_ec2_security_group.this_security_group_id
+    },
+    {
+      from_port                = 10020
+      to_port                  = 10020
+      protocol                 = "tcp"
+      source_security_group_id = module.abbyy_doc_ocr_ec2_security_group.this_security_group_id
+    },
+    {
+      rule                     = "http-80-tcp"
+      source_security_group_id = module.abbyy_doc_ocr_internal_alb_security_group.this_security_group_id
+    }
+  ]
+  number_of_computed_ingress_with_source_security_group_id = 3
+
+  egress_rules = ["all-all"]
+}
+
+# ------------------------------------------------------------------------------
+# ABBYY Document Capture & OCR CloudWatch
+# ------------------------------------------------------------------------------
+
+resource "aws_cloudwatch_log_group" "abbyy_doc_ocr" {
+  for_each = local.abbyy_doc_ocr_cw_logs
+
+  name              = each.value["log_group_name"]
+  retention_in_days = lookup(each.value, "log_group_retention", var.default_log_group_retention_in_days)
+  kms_key_id        = lookup(each.value, "kms_key_id", data.aws_kms_key.logs.arn)
+
+  tags = merge(
+    local.default_tags,
+    map(
+      "Name", "${var.application}-abbyy-doc-ocr-server",
+      "ServiceTeam", var.ServiceTeam
+    )
+  )
+}
+
+# ------------------------------------------------------------------------------
+# ABBYY Document Capture & OCR EC2
+# ------------------------------------------------------------------------------
+
+resource "aws_instance" "abbyy_doc_ocr_ec2" {
+  count                  = var.abbyy_doc_ocr_ec2_count
+  ami                    = var.abbyy_doc_ocr_ami
+  key_name               = aws_key_pair.abbyy_doc_ocr_keypair.key_name
+  instance_type          = var.abbyy_doc_ocr_ec2_instance_size
+  monitoring             = var.monitoring
+  get_password_data      = var.get_password_data
+  subnet_id              = sort(data.aws_subnet_ids.application.ids)[count.index]
+  vpc_security_group_ids = [module.abbyy_doc_ocr_ec2_security_group.this_security_group_id, data.aws_security_group.nagios_shared.id, data.aws_security_group.rdp_shared.id]
```

Comment:
You can remove Nagios from this and future work as its being replaced.
Dont forget to remove the data lookup.

Reply:
Nagios references removed.